Rapid7 InsightVM Cloud Plugin Update (#914)
rapid7/insightconnect-plugins: rapid7_insightvm_cloud/help.md

> 

Reviewer: Up above, add a bit more details to descript:
`InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the v4 cloud API to view assets, retrieve scan results and start scans. With an InsightVM scan engine paired to the platform, take advantage of triggering scans on assets in your environment along with performing robust asset searches to lookup and make actionable the asset and vulnerability data in your environment.`

# Key Features

> * View assets

Reviewer 2: @sengleson-r7 Do you have good language around use-cases?

> * View assets
> * Start scans
> * Get scan results

Reviewer: How about something like:
```suggestion
* Perform scan management functionality including starting and checking the status of scans
* Perform asset searches leveraging flexible asset and vulnerability filters
* Retrieve individual asset information
```

# Requirements

> *Requires an InsightVM API Key

Reviewer 2: ```suggestion
* Requires an InsightVM API Key
```

@zyoutz-r7 anything else to call out here for the IVM Cloud?

> *Requires an InsightVM API Key

Reviewer: ```suggestion
* Requires an InsightVM API Key
* InsightVM Scan Engine(s) paired to the Insight Platform
```

Reviewer: Also under resources lets add:
[Scan Engine Pairing](https://docs.rapid7.com/insightvm/scan-engine-management-on-the-insight-platform/#how-to-pair-your-scan-engines-to-the-insight-platform)

# Version History

> * 2.0.0 - Fixed issues in actions and help doc

Reviewer 2: ```suggestion
* 2.0.0 - Fix issue where string data-type should be date in actions | Update docs
```
